doc/lua_api.txt

```diff
@@ -4743,6 +4743,12 @@ The Biome API is still in an experimental phase and subject to change.
     --  ^ Flags for schematic decorations.  See 'Schematic attributes'.
         rotation = "90" -- rotate schematic 90 degrees on placement
     --  ^ Rotation can be "0", "90", "180", "270", or "random".
+        y_offset = 1,
```

Comment:
Wait. If the default offset is 1, it means the schematic is above 1 node relative to the place_node. This will break all existing schematics by moving them up 1 node. So shouldn't the default offset be 0?

Reply:
The default is zero, 1 is used here as an example, because other parameters are documented with example-type values.

Reply:
But, i should make the default value clearer.

Reply:
Oops. I thought the “parameter = value” syntax always showed the default value. At least this seems to be the case with most things in `lua_api.txt`.